### ISSUE TITLE
Add kernel modules to support the NFS4 clients

### ISSUE THREAD

**Issue by mikhail-vl** [balena-os/balena-jetson-orin#11]
We cannot use the NFS client to share files between containers.

```
nginx  mount.nfs: Protocol not supported
grafana  mount.nfs: Protocol not supported
```

Could you please add NFS4/3 client modules similar to Jetson AGX and Xavier? https://github.com/balena-os/balena-jetson/pull/354

The concept is explained in the article: https://www.balena.io/blog/using-nfs-server-to-share-external-storage-between-containers-balena/

**Comment by acostach** [comment]
Adding them in https://github.com/balena-os/balena-jetson-orin/pull/9

**Comment by mikhail-vl** [comment]
@acostach Thank you, I have built our own image base on #9 and flashing it now to test.

**Comment by acostach** [comment]
Perfect, let me know if those configs are fine

**Comment by mikhail-vl** [comment]
@acostach 

  I did my own build yesterday based on the `master` and today based on #9. Looking at the forum and around, I followed three steps (Build, Configure and Flash). After starting device provisioned itself in the balenaStaging and downloaded the release.
  Everything seems to be fine, but I can't connect to the Host OS, and it's unable to start. Any pointers on what I am missing?

1. Build

```
./balena-yocto-scripts/build/barys
```

2. Configure

```
BALENARC_BALENA_URL=balena-staging.com  /app/balena-cli/balena os configure --fleet orins balena-image-jetson-agx-orin-devkit-20221026121045.rootfs.balenaos-img 
```

3. Flash

```
$ ./bin/cmd.js -f ../balena-image-jetson-agx-orin-devkit-20221026121045.rootfs.balenaos-img -m jetson-agx-orin-devkit
```

![Screen Shot 2022-10-26 at 08 53 59](https://user-images.githubusercontent.com/47795110/198031780-6d4f3e2f-a30c-4419-ba76-e97ebd54c514.png)
